mne/io/nicolet/nicolet.py

```diff
@@ -204,3 +206,52 @@ def __init__(self, input_fname, ch_type, montage=None, eog=(), ecg=(),
     def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
         """Read a chunk of raw data"""
         _read_segments_file(self, data, idx, fi, start, stop, cals, mult)
+
+
+def read_nicolet_annotations(fname, record_id):
```

Comment:
the data you have access to do not really correspond to a public format. We should not push this code to mne-python.